boms/cloud-oss-bom/pom.xml

```diff
@@ -48,19 +48,19 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <guava.version>31.0.1-jre</guava.version>
-    <google.cloud.bom.version>0.169.0</google.cloud.bom.version>
-    <google.cloud.core.version>2.4.0</google.cloud.core.version>
-    <io.grpc.version>1.44.0</io.grpc.version>
-    <http.version>1.41.2</http.version>
-    <protobuf.version>3.19.3</protobuf.version>
+    <google.cloud.bom.version>0.170.0</google.cloud.bom.version>
+    <google.cloud.core.version>2.5.6</google.cloud.core.version>
+    <io.grpc.version>1.44.1</io.grpc.version>
+    <http.version>1.41.4</http.version>
+    <protobuf.version>3.19.4</protobuf.version>
     <!-- We don't use gax-bom because it includes the artifacts with 'testlib' classifier.
         When updating gax.version, update gax.httpjson.version too. -->
-    <gax.version>2.11.0</gax.version>
-    <gax.httpjson.version>0.96.0</gax.httpjson.version>
-    <auth.version>1.4.0</auth.version>
-    <api-common.version>2.1.3</api-common.version>
-    <common.protos.version>2.7.2</common.protos.version>
-    <iam.protos.version>1.2.1</iam.protos.version>
+    <gax.version>2.12.2</gax.version>
+    <gax.httpjson.version>0.97.2</gax.httpjson.version>
```

Comment:
https://search.maven.org/artifact/com.google.api/gax-bom/2.12.2/pom has 0.97.2 for gax-httpjson